iocage/lib/Jail.py

```diff
@@ -465,8 +467,11 @@ def start(
         self._write_hook_script(
             "poststart",
             self._wrap_hook_script_command_string([
+                "set -u",
+                "set -e",
+                "set -x",
                 "/bin/echo running exec.started hook on the host",
-                f"/bin/sh {self.get_hook_script_path('started')}",
+                f"/bin/sh {self.get_hook_script_path('started')} 2>&1",
```

Comment:
can you briefly explain why we cannot get `stdin` & `stdout` at forwarded to us?

Reply:
We iterate over stdout to get async access. When stdout is written the execution is stuck because nobody consumes the arriving data. We can solve it with threading, but that can be achieved in a later step. In the meanwhile we need to combine stderr and stdout.